testing/tests/DevExpress.exporter/jspdfParts/jspdfTestHelper.js

```diff
@@ -6,6 +6,24 @@ class JSPdfDataGridTestHelper {
         this.jsPDFDocument = jsPDFDocument;
     }
 
+    checkCustomizeCell(eventArgs, expectedCells, callIndex) {
+        assert.ok(eventArgs.gridCell);
```

Comment:
Why we do not check the actual gridCell param as we do this in exceljs export? We should to be convinced that it necessary grid cell that according to source DataGrid cell.

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/15039/commits/961ded0817b3666c0a70b4d9cb2f065cfecaac0a

Reply:
fixed